lib/loader/loader.c

```diff
@@ -270,25 +287,8 @@ struct evmc_instance* evmc_load_and_configure(const char* config,
 
     while (strlen(options) != 0)  // TODO: Can be optimized as *options != 0 because MSVC sucks.
     {
-        char* option = options;
-        char* opt_sep = strchr(options, ',');
-        if (opt_sep)
-        {
-            *opt_sep = '\0';
-            options = opt_sep + 1;
-        }
-        else
-            options = "";
-
-
-        char* kv_sep = strchr(option, '=');
-        const char* name = option;
-        char* value = "";  // Set value to empty to handle the "...,option,..." case.
-        if (kv_sep)
-        {
-            *kv_sep = '\0';
-            value = kv_sep + 1;
-        }
+        char* value = get_prefix(&options, ',');
```

Comment:
Should add a comment here why parsing is done "in reverse" (e.g. to support options without a value)